addons/GearSwap/libs/ccConfig.lua

```diff
@@ -0,0 +1,30 @@
+-- Jobs you want to execute with, recomment put all active jobs you have lua for will look for <job>.lua or <playername>_<job>.lua files
+ccjobs = { 'BLM', 'BLU', 'BRD', 'BST', 'DRG', 'GEO', 'MNK', 'NIN', 'PLD', 'RDM', 'RNG', 'RUN', 'SAM', 'SCH', 'SMN', 'THF', 'WAR', 'WHM' }
+-- Put any items in your inventory here you don't want to show up in the final report
+-- recommended for furniture, food, meds, pop items or any gear you know you want to keep for some reason
+-- supports lua regex syntax, put each separate one as its own entry in the table
+-- see: https://www.lua.org/pil/20.2.html
+ccignore = { "Rem's Tale*", "Storage Slip %d+", "Deed of*", "%a+ Virtue", "Dragua's Scale",
+			"Glittering Yarn", "Dim. Ring*", "Cupboard", ".*VCS.*", ".*Abjuration.*", "%a+ Organ",

```

Comment:
Use spaces instead of tabs please. Superficially it's not too important, but we've been treating it as a policy. You can configure most editors to use spaces when you press the tab key.

The same goes for the other file.